src/main/resources/static/views/login/login.html

```diff
@@ -45,7 +45,7 @@ <h1 class="portalheader">
                                             <div class="row">
                                                 <div class="col-lg-12">
                                                     <div class="text-center">
-                                                        <a href="https://phpoll.com/recover" tabindex="5" class="forgot-password">Forgot Password?</a>
+                                                        <a href="#" tabindex="5" class="forgot-password" id="noaccount-form-link">Don't have an account?</a>

```

Comment:
This is good. But what if they have an account already but they want to just recover their password. I believe we should still have the forgot password there.

Reply:
I agree, however, I think it's out of scope for this project. If we decide to implement password recovery we can always add a link :)